modules/installation-about-custom-azure-vnet.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-vnet.adoc
+
+[id="installation-about-custom-azure-vnet_{context}"]
+= About reusing a VNet for your {product-title} cluster
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VNet in Microsoft Azure. If you do, you must also use existing subnets within the VNet and routing rules.
+
+By deploying {product-title} into an existing Azure VNet, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VNet yourself.
+
+[IMPORTANT]
+====
+Use of an existing VNet requires the use of the updated Azure Private DNS (preview) feature. See link:https://azure.microsoft.com/en-us/updates/announcing-preview-refresh-for-azure-dns-private-zones-2/[Announcing Preview Refresh for Azure DNS Private Zones] for more information about the limits of this feature.
+====
+
+[id="installation-about-custom-azure-vnet-requirements_{context}"]
+== Requirements for using your VNet
+
+The installation program will no longer create the following components:
```

Comment:
will no longer create -> no longer creates

Like in your AWS PR. Though I admit I don't understand what this means. Do you mean as compared to 4.2? Or it doesn't create these only in the even that you're using your own VNet (or VPC?)

Reply:
Thank you for pointing out that this wasn't clear enough. I'm thinking of using this text instead:

> When you deploy a cluster by using an existing VNet, you must perform additional network configuration before you install the cluster. In installer-provisioned infrastructure clusters, the installer usually creates the following components, but it does not create them when you install into an existing VNet: